compiler/qsc_linter/src/lints/ast.rs

```diff
@@ -0,0 +1,38 @@
+use super::{declare_lint, push_lint};
+use crate::linter::{ast::AstLintPass, Lint, LintLevel};
+use qsc_ast::ast::{BinOp, ExprKind, Lit};
+
+declare_lint!(
+    DoubleParens,
+    LintLevel::Warning,
+    "unnecesary double parentheses"
+);
+declare_lint!(
+    DivisionByZero,
+    LintLevel::Allow,
```

Comment:
Should division by zero be an error?

Reply:
I'm not sure. When I showed it on the demo I think Dmitry said something about it being possible to do in Q#. I just left it there in case the users wanted to enable that lint manually.